frontend/src/app/commonComponents/Header.tsx

```diff
@@ -39,7 +45,10 @@ const Header = () => {
 
   const onFacilitySelect = (e: React.FormEvent<HTMLSelectElement>) => {
     const id = (e.target as HTMLSelectElement).value;
-    dispatch(updateFacility(facilities.find((f) => f.id === id)));
+    // dispatch(updateFacility(facilities.find((f) => f.id === id)));
+    // props.updateFacilityId(id);
+    // history.push(`${window.location.pathname}?facility=${id}`);
```

Comment:
this is the ideal implementation to not trigger a hard refresh, but in demo, it appends the pathname to the existing path.

ie: if the path is `/app/queue?facility=123`, then this line would change it to: `/app/app/queue?facility=456`. Couldn't figure out how to debug this